opm/parser/eclipse/EclipseState/Tables/VFPProdTable.cpp

```diff
@@ -377,32 +380,39 @@ VFPProdTable::GFR_TYPE VFPProdTable::getGFRType(std::string gfr_string) {;
 
 
 
-VFPProdTable::ALQ_TYPE VFPProdTable::getALQType(std::string alq_string) {
-    if (alq_string == "GRAT") {
-        return ALQ_GRAT;
-    }
-    else if (alq_string == "IGLR") {
-        return ALQ_IGLR;
-    }
-    else if (alq_string == "TGLR") {
-        return ALQ_TGLR;
-    }
-    else if (alq_string == "PUMP") {
-        return ALQ_PUMP;
-    }
-    else if (alq_string == "COMP") {
-        return ALQ_COMP;
-    }
-    else if (alq_string == "BEAN") {
-        return ALQ_BEAN;
-    }
-    else if (alq_string == " ") {
+VFPProdTable::ALQ_TYPE VFPProdTable::getALQType(std::shared_ptr<const DeckItem> item) {
+    if (item->defaultApplied(0))
```

Comment:
Hehe - I realize I am in a quite desperate minority here, but quite frankly I really don't like those braces - I think they make the code uglier and less readable. But as long as the rest of the method is written with braces I'll succumb.